toil/src/cgcloud/toil/toil_cluster.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from cgcloud.toil.toil_box import ToilBox, ToilLeader
+from cgcloud.mesos.mesos_cluster import CreateMesosCluster
+from subprocess import check_call
+
+log = logging.getLogger( __name__ )
+shared_dir="/home/ubuntu/shared/"
+class CreateToilCluster(CreateMesosCluster):
+    def __init__( self, application ):
+        super( CreateToilCluster, self ).__init__( application )
+        self.option( '--dir_to_copy', default=None,
```

Comment:
What do you think about using ``--shared-dir`` instead?

In either case, it should be using dashes not underscores, IIRC.